### PR TITLE
[ci] Store capybara screenshots as artifacts in CircleCI

### PR DESCRIPTION
**Pull request by ChrisBr**
for better debugging.

### CODE REVIEW

File: src/api/spec/support/capybara.rb [hunk 1]
```diff
@@ -16,15 +16,20 @@
 
 # Automatically save the page a test fails
 RSpec.configure do |config|
+  config.before(:suite) do
+    html_files = Dir.glob(File.join(Capybara.save_path, '*.html'))
+    png_files = Dir.glob(File.join(Capybara.save_path, '*.png'))
+    FileUtils.rm(html_files + png_files)
```
Comment (hennevogel) [inline src/api/spec/support/capybara.rb]: :thought_balloon: why not just remove/store all files?

Comment (ChrisBr) [reply src/api/spec/support/capybara.rb]: We can certainly do that, however, I did not want to change the behavior as I don't know what else gets stored in the capybara directory

Comment (hennevogel) [reply src/api/spec/support/capybara.rb]: Just this...

**Comment by ChrisBr** [comment]
@hennevogel updated

**Comment by ChrisBr** [comment]
Only flaky dod feature test failed, merging ....

**Pull request merged**